README.md

```diff
@@ -147,4 +145,43 @@ with f:
     f.search(doc_query, parameters={'filter': filter_})
 ```
 
+### Limit results
+
+In some cases, you will want to limit the total number of retrieved results. `QdrantIndexer` uses the `limit` argument 
+from the `match` function to set this limit. Note that when using `shards=N`, the `limit=K` is the number of retrieved results for **each shard** and total number of retrieved results is `N*K`. By default, `limits` is set to `20`. For more information about shards, please read [Jina Documentation](https://docs.jina.ai/fundamentals/flow/topology/#partition-data-by-using-shards)
+
+```python
+f =  Flow().add(
+    uses='jinahub+docker://QdrantIndexer',
+    uses_with={'match_args': {'limit': 10}})
+```
+
+### Configure other search behaviors
+
+You can use `match_args` argument to pass arguments to the `match` function as below. The match function will be called
+during `/search` endpoint.
+
+```python
+f =  Flow().add(
+     uses='jinahub+docker://QdrantIndexer',
+     uses_with={
+         'match_args': {
+            'limit': 5,
+})
+```
+
+- For more details about overriding configurations, please refer to [here](https://docs.jina.ai/fundamentals/executor/executor-in-flow/#special-executor-attributes).
+- You can find more about `filtering`  at Qdrant [here](https://qdrant.tech/documentation/filtering/)
+
+### Configure the Search Behaviors on-the-fly
+
+**At search time**, you can also pass arguments to config the `match` function. This can be useful when users want to query with different arguments for different data requests. For instance, the following codes query with a custom `limit` in `parameters` and only retrieve the top 100 nearest neighbors. This will overwrite existing `match_args` if defined when initialized.
```

Comment:
```suggestion
**At search time**, you can also pass arguments to parametrize the `match` function. This can be useful when users want to query with different arguments for different data requests. For instance, the following code makes query with a custom `limit` in `parameters` and only retrieve the top 100 nearest neighbors. This will override existing parameters in `match_args` if defined during Executor intialization.
```